src/zoo.js

```diff
@@ -12,8 +12,9 @@ eslint no-unused-vars: [
 
 const data = require('./data');
 
-function animalsByIds(ids) {
-  // seu código aqui
+function animalsByIds(...ids) {
+  const animals = data.animals.filter(elem => ids.find( id => id == elem.id));
+  return animals;
 }
```

Comment:
Código limpo e fácil de entender. Também poderia trocar o _find()_, pelo _includes()_. Depois de filtrar o animal pelo ID, basta verificar se ele contém no _elem.id_.

```suggestion
}const animals = data.animals.filter(elem => ids.includes(elem.id);
```

Reply:
otima sugestão! vou aderir! :)